website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -42,7 +42,7 @@ $ operator-sdk create api -h
 Next, we will create a `Memcached` API.
 
 ```sh
-$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --generate-role
+$ operator-sdk create api --group cache --version v1 --kind Memcached --generate-role
```

Comment:
IMO if we do this update we need to do across the project which means: 
- update ansible/helm docs
- update ansible/helm code samples that generate operator-sdk/testdata/
- update the e2e tests 

Then, would be nice also update the go project to work with v1 which means that it would be blocked by https://github.com/operator-framework/operator-sdk/issues/3580. So, wdyt about we raise an issue to track this need and close this PR?